scripts/build_updates_img.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# A script that wraps make install to create an updates.img suitable for anaconda.
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+UPDATE_DIR=$(mktemp -d)
+pushd $PROJECT_ROOT
+
+rm ./updates.img 2>/dev/null
+
+# Install files to the temporary build dir
+export DESTDIR=$UPDATE_DIR
+make -e install
```

Comment:
Run e.g. following commands before `make -e install` to be sure all required files are present:

    make clean
    make

Reply:
It would be necessary to use `python{version} ./setup.py build` and not `make` to support various versions of Python.